File: example/ApiTest/Python/chats/3_list_chats.py
import requests

url = "http://127.0.0.1/api/v1/chats?name=api_create_dataset&id=8a85ab34ad5311ef98b00242ac120003"

payload = {}
headers = {
  'Authorization': 'Bearer ragflow-hjNzA4ODI4YWM5MTExZWY5YzUyMDI0Mm'
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)

File: example/ApiTest/JavaScript/chats/3_list_chats.js
const myHeaders = new Headers();
myHeaders.append("Authorization", "Bearer ragflow-hjNzA4ODI4YWM5MTExZWY5YzUyMDI0Mm");

const requestOptions = {
  method: "GET",
  headers: myHeaders,
  redirect: "follow"
};

fetch("http://127.0.0.1/api/v1/chats?name=api_create_dataset&id=8a85ab34ad5311ef98b00242ac120003", requestOptions)
  .then((response) => response.text())
  .then((result) => console.log(result))
  .catch((error) => console.error(error));